util/topgen/lib.py

```diff
@@ -3,16 +3,17 @@
 # SPDX-License-Identifier: Apache-2.0
 
 import logging as log
+import re
+import sys
+from collections import OrderedDict
 from copy import deepcopy
 from pathlib import Path
-from collections import OrderedDict
-import hjson
-import sys
 
-import re
+import hjson
 
 # Ignore flake8 warning as the function is used in the template
-from .intermodule import im_defname, im_netname, im_portname  # noqa : F401
+from .intermodule import im_defname  # noqa : F401
+from .intermodule import find_otherside_module, im_netname, im_portname
```

Comment:
@eunchan 
I can't make both flake8 and yapf happy for these 2 lines
now yapf is passing, flake8 fails.

if I change to below, yapf will fail
```
from .intermodule import im_defname  # noqa : F401
from .intermodule import find_otherside_module, im_netname, im_portname # noqa : F401
```

Reply:
Hum. then, maybe you can waive by putting yapf:disable?

Reply:
Good idea. I added `isort:skip`